src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs

```diff
@@ -1334,6 +1334,33 @@ private void WriteResourceAssemblies()
                     {
                         WriteMetadata(MetadataKeys.AssetType, "resources");
                         string locale = asset.Properties["locale"];
+                        // Locales from packages can be free-form, so we normalize them to the standard
+                        // forms here. If the locale is mixed-case, that can cause issues on case-sensitive
+                        // file systems when the locale-specific assets are copied.
+                        try
+                        {
+                            var normalizedLocale = System.Globalization.CultureInfo.GetCultureInfo(locale).Name;
+                            if (normalizedLocale != locale)
+                            {
+                                _task.Log.LogWarning(
+                                    string.Format(Strings.PackageContainsIncorrectlyCasedLocale,
```

Comment:
You don't need the call to `string.Format` here, `LogWarning` takes a list of args that it will pass to `string.Format`